src/Features/CSharp/Portable/ConvertTupleToStruct/CSharpConvertTupleToStructCodeRefactoringProvider.cs

```diff
@@ -32,5 +34,21 @@ internal class CSharpConvertTupleToStructCodeRefactoringProvider :
         public CSharpConvertTupleToStructCodeRefactoringProvider()
         {
         }
+
+        protected override ArgumentSyntax GetArgumentWithChangedName(ArgumentSyntax argument, string name)
+        {
+            return argument.WithNameColon(ChangeName(argument.NameColon, name));
+        }
+
+        private static NameColonSyntax? ChangeName(NameColonSyntax? nameColon, string name)
```

Comment:
nit: i'd just call this `WithName`

Reply:
I don't love `ChangeName`, but `WithName` looks odd not being an extension method, so going to leave this.